esp/services/ws_topology/ws_topologyService.cpp

```diff
@@ -798,6 +853,9 @@ void CWsTopologyEx::readTpLogFileRequest(IEspContext &context, const char* fileN
     readLogReq.prevPage = -1;
     readLogReq.nextPage = -1;
     readLogReq.lastHours = 0;
+    readLogReq.logfields = 0;//legacy log format
+    readLogReq.logDatePos = 9;//legacy log format
+    readLogReq.logTimePos = 20;//legacy log format
```

Comment:
See the comment in Live 574 regarding using MSGFIELD_LEGACY. The legacy column number shouldn't be hard coded.